src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -95,6 +95,10 @@ export class MongoStorageAdapter {
     });
   }
 
+  purgeCollection(name: string) {
```

Comment:
We already have `deleteObjectsByQuery` which will remove all objects if you pass `{}` as the query. Can you use that instead?